src/routes/[dictionaryId]/contributors.svelte

```diff
@@ -52,7 +52,7 @@
   function writeIn() {
     const name = prompt(`${$_('speakers.name', { default: 'Name' })}?`);
     if (name) {
-      addOnline(`dictionaries/${dictionaryId}/writeInCollaborators`, { name });
```

Comment:
When write in a collaborator, I'm going to use "add" which is the normal Firestore add method. It first saves to cache and then to server, as that will give them faster feedback. I don't await just in case the server write hangs and has to be done later from the cache.